mirdata/datasets/dagstuhl_choirset.py

```diff
@@ -0,0 +1,411 @@
+volume    = {3},
+number    = {1},
+year      = {2020},
+pages     = {98--110},
+publisher = {Ubiquity Press},
+doi       = {10.5334/tismir.48},
+url       = {http://doi.org/10.5334/tismir.48},
+url-demo  = {https://www.audiolabs-erlangen.de/resources/MIR/2020-DagstuhlChoirSet}
+}
+"""
+
+REMOTES = {
+    "full_dataset": download_utils.RemoteFileMetadata(
+        filename="DagstuhlChoirSet_V1.2.3.zip",
+        url="https://zenodo.org/record/4618287/files/DagstuhlChoirSet_V1.2.3.zip?download=1",
+        checksum="82b95faa634d0c9fc05c81e0868f0217",
+        unpack_directories=["DagstuhlChoirSet_V1.2.3"],
+    ),
+}
+
+DOWNLOAD_INFO = """
+Downloading dataset from Zenodo (5.1 GB)...
+"""
+
+LICENSE_INFO = """
+Creative Commons Attribution 4.0 International
+"""
+
+
+class Track(core.Track):
+    """Dagstuhl ChoirSet Track class
+
+    Args:
+        track_id (str): track id of the track
+
+    Attributes:
+        audio_paths (list): paths to audio files
+        f0_paths (list): paths to F0-trajectories
+        score_paths (list): paths to time-aligned score representation
+        track_id (str): track id
+
+    Cached Properties:
+        f0 (annotations.F0Data): returns specified F0-trajectory
+        score (annotations.NoteData): returns time-aligned score representation
+    """
+
+    def __init__(self, track_id, data_home, dataset_name, index, metadata):
+
+        super().__init__(
+            track_id=track_id,
+            data_home=data_home,
+            dataset_name=dataset_name,
+            index=index,
+            metadata=metadata,
+        )
+
+        self.audio_paths = [
+            self.get_path(key)
+            for key in self._track_paths
+            if "audio" in key
+            if self.get_path(key)
+        ]
+
+        self.f0_paths = [
+            self.get_path(key)
+            for key in self._track_paths
+            if "f0" in key
+            if self.get_path(key)
+        ]
+
+        self.score_paths = [
+            self.get_path(key)
+            for key in self._track_paths
+            if "score" in key
+            if self.get_path(key)
+        ]
+
+    def f0(self, mic="lrx", ann="crepe"):
+        """Get F0-trajectory of specified type extracted from specified microphone
+        Args:
+            mic (str): Identifier of the microphone ("dyn", "hsm", or "lrx")
+            ann (str): Identifier of the annotation ("crepe", "pyin", or "manual")
+
+        Returns:
+            F0Data Object: F0-trajectory
+        """
+        if mic.lower() not in ["dyn", "hsm", "lrx"]:
+            raise ValueError("mic={} is invalid".format(mic))
+
+        if ann.lower() not in ["crepe", "pyin", "manual"]:
+            raise ValueError("ann={} is invalid".format(ann))
+
+        f0_path = [
+            s
+            for s in self.f0_paths
+            if mic.lower() in s.lower()
+            if ann.lower() in s.lower()
+        ]
+
+        if not f0_path:
+            return None
```

Comment:
These two lines can be removed - see my comment on line 309

Reply:
done!